Makefile

```diff
@@ -231,7 +231,7 @@ install-git-hooks:
 # Unit Tests
 ###############################################################################
 ## Run the unit tests.
-ut: run-k8s-apiserver build $(BIN)/host-local
+ut: run-k8s-apiserver run-k8s-controller build $(BIN)/host-local
```

Comment:
I think this is a bit awkward, because the way it is ordered now we're going to (ignore the etcd bits):

- Stop any running apiserver
- Run the apiserver
- Stop any k8s controller
- Stop any running apiserver
- Run the apiserver
- Run the k8s controller

I think we should clean this up so we're not starting / stopping the k8s API so many times.